cli-ext/Makefile

```diff
@@ -30,6 +32,11 @@ upx-compress: $(BUILDDIR)
 %.sha256: %
 	shasum -a 256 $< > $@
 
+.PHONY: build
+build:
+	npm run build
+	$(MAKE) $(BINARY_ASSET_CHECKSUMS)
```

Comment:
Generating the SHAs for the cli-ext binaries just in case if we need to use them for verification in the future. (original thought was to use it this PR, but we have decided just do a overwrite of binary whenever cli-ext is needed in cli for now)